releasenotes/notes/dsd-container-b2039d8e0e510fbf.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    DogstatsD origin detection on UDP supports container entities in addition to Kubernetes Pods.
```

Comment:
Could we rephrase this to not talk about UDP but about supporting a new field in dogstatsd (the new field is unrelated to UDP) ?